tests/ctst/steps/utils/utils.ts

```diff
@@ -374,6 +392,60 @@ async function addTransitionWorkflow(this: Zenko, location: string, enabled = tr
     }
 }
 
+function getReplicationLocationConfig(world: Zenko, location: string): {
+        storageClass: string;
+        destinationBucket: string;
+        locationType: string;
+    } {
+    switch (location) {
+    case 'awsbackendmismatch':
+        return {
+            storageClass: world.parameters.AwsBackendDestinationReplicationLocation,
+            destinationBucket: world.parameters.AwsReplicationBucketName,
+            locationType: world.parameters.AwsReplicationLocationType,
+        };
+    default:
+        throw new Error(`Unsupported replication location: '${location}'`);
+    }
+}
+
+async function putBucketReplication(
+    this: Zenko,
+    srcBucket: string,
+    replicationLocation: string
+) {
+    const { storageClass, destinationBucket } = getReplicationLocationConfig(this, replicationLocation);
+
+    this.resetCommand();
+    this.addCommandParameter({ bucket: srcBucket });
+    this.addCommandParameter({
+        replicationConfiguration: JSON.stringify({
+            Role: 'arn:aws:iam::root:role/s3-replication-role',
+            Rules: [
+                {
+                    Prefix: '',
+                    Destination: {
+                        Bucket: `arn:aws:s3:::${destinationBucket}`,
```

Comment:
Not sure this is really used in Zenko's multi-backend replication (since the target bucket is actually part of the storage location definition), but [Artesca documentation](https://documentation.scality.com/Artesca/4.0.1/data_management/bucket_operations/replication_workflow/create_a_replication_workflow.html) mentions to set `srcBucket` here:

```suggestion
                        Bucket: `arn:aws:s3:::${srcBucket}`,
```

(It will however be relevant for CRR which allows replicating to "any" bucket for the CRR location/cluster)